gm4_bookshelf_nametags/data/gm4_bookshelf_nametags/functions/evaluate/get_pos.mcfunction

```diff
@@ -0,0 +1,9 @@
+# spawn marker to get player position
+# @s = marker
+# at unspecified
+# run from evaluate/motion
+
+tp @s @p[tag=gm4_bookshelf_nametags_target]
+data modify storage gm4_bookshelf_nametags:temp Motion set from entity @s Motion
```

Comment:
You're technically reading player NBT twice here, once for `Motion` and once for `Rotation`. It could be cheaper to read it once and then extract `Motion` and `Rotation`, I don't fully remember though.

Reply:
I've changed this, but it does require me to read the entire marker NBT into storage so I'm not sure if it's actually faster